thirdparty/jquery-entwine/jquery.entwine.js

```diff
@@ -1457,26 +1393,7 @@ catch (e) {
 
 /* src/jquery.entwine.events.js */
 
-(function($) {	
-
-	/** Taken from jQuery 1.5.2 for backwards compatibility */
-	if ($.support.changeBubbles == undefined) {
```

Comment:
`$.support` [is deprecated](https://api.jquery.com/jquery.support/) but we don't need this anyway - it's to support old browsers we don't care about.